DialogHost.Avalonia/DialogHost.axaml.cs

```diff
@@ -521,26 +521,26 @@ public DialogClosingEventHandler? DialogClosingCallback {
         return instance.ShowCore(content, openedEventHandler, closingEventHandler);
     }
 
-    /// <summary>Close a modal dialog.</summary>
-    /// <param name="dialogIdentifier">of the instance where the dialog should be closed. Typically this will match an identifier set in XAML.</param>
-    public static void Close(string? dialogIdentifier)
-        => Close(dialogIdentifier, null);
+    ///// <summary>Close a modal dialog.</summary>
+    ///// <param name="dialogIdentifier">of the instance where the dialog should be closed. Typically this will match an identifier set in XAML.</param>
+    //public static void Close(string? dialogIdentifier)
+    //    => Close(dialogIdentifier, null);
```

Comment:
This would be a breaking change. Probably this is needs to use fallback if there is an single dialog open currently.